src/Controller/LuckyController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+// src/Controller/LuckyController.php
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class LuckyController extends AbstractController
+{
+     /**
+      * @Route("/lucky/number")
+      */
+    public function number(): Response
+    {
+        $number = random_int(0, 100);
+
+        return $this->render('lucky/number.html.twig', [
+            'number' => $number,
+        ]);
+    }
+}
```

Comment:
End of line

Reply:
I've updated the commit and fixed this issue, please check out the new commit content.